18 спринт

Gammius opened the pull request.

Gammius committed:
18 спринт

ArturNurtdinov commented on app/src/main/java/com/example/playlist_maker/presentation/mediaLibrary/activity/FavoritesFragment.kt:
```diff
@@ -0,0 +1,48 @@
+        fun newInstance(): FavoritesFragment {
+            return FavoritesFragment().apply {
+                arguments = Bundle().apply {
+                }
```
Создание пустого `Bundle` можно убрать, чтобы не создавать лишние объекты

Gammius replied:
убрал

ArturNurtdinov commented on app/src/main/java/com/example/playlist_maker/presentation/mediaLibrary/view_model/MediaLibraryViewModel.kt:
```diff
@@ -0,0 +1,19 @@
+
+    init {
+        _mediaLibrary.value = MediaLibraryActivityState(
+            tabTitles = listOf("Избранные треки", "Плейлисты"),
```
Строчки необходимо вынести в ресурсы приложения. Для этого экрана не обязательно нужно табы отдавать через вьюмодель и лайвдату, так как табы зафиксированы и не меняются, поэтому можно только в активити настроить отображение TabLayout без использования вьюмодели

Gammius replied:
убрал из вьюмодели
вынес в ресурсы

ArturNurtdinov reviewed (commented):
В тёмной теме цвет бэкграунда экрана медиатеки не соответствует макетам (цвет TabLayout отличается от цвета бэкграунда), цвет должен быть #1A1B22:
![image](https://github.com/user-attachments/assets/028c32ba-a096-4ad8-ab8e-d7eab5c8f986)
Макет:
![image](https://github.com/user-attachments/assets/386a4d31-48a1-4459-b34f-231c9d171d06)

Gammius committed:
18 спринт
критические:
исправил конечно, но не знаю как еще
в теме бг перекрывает TabLayout
стиль к активити не работает
бг в коде указываешь , все рано тема перекрывает.
не критическую позже исправлю

Gammius committed:
18 спринт
пропустил критическую. доделал
